### PR TITLE
Update comfoconnect.py

### PR DESCRIPTION
**Pull request by jonesPD**
Improved stability by catching many (all?) of the exceptions.

### CODE REVIEW

**Comment by michaelarnauts** [comment]
It looks like I've missed this one, but I'm a bit hesitant to merge since it seems to have changed quite some things.

File: pycomfoconnect/comfoconnect.py [hunk 1]
```diff
@@ -123,19 +142,22 @@ def connect(self, takeover=False):
...
 
         except PyComfoConnectNotAllowed:
             raise Exception('Could not connect to the bridge since the PIN seems to be invalid.')
 
         except PyComfoConnectOtherSession:
             raise Exception('Could not connect to the bridge since there is already an open session.')
 
-        except:
+        except OSError:
+            print("Unexpected error in connect: ", sys.exc_info()[0])
```
Comment (michaelarnauts) [inline pycomfoconnect/comfoconnect.py:154]: No prints please.

File: pycomfoconnect/comfoconnect.py [hunk 2]
```diff
@@ -212,19 +247,29 @@ def _command(self, command, params=None, use_queue=True):
         self._reference += 1
 
         # Send the message
-        self._bridge.write_message(message)
+        try: 
+            reply = self._bridge.write_message(message)
+        
+        except OSError:
+            print("Unexpected error in _command._bridge.write_message: ", sys.exc_info()[0])
```
Comment (michaelarnauts) [inline pycomfoconnect/comfoconnect.py:254]: Same

**Comment by michaelarnauts** [comment]
I prefer to use `logging` for the debug messages.

Use a _LOGGER = logging.getLogger("pycomfoconnect") at the top and then you can use _LOGGER.info() or _LOGGER.debug() to log messages.

**Comment by jonesPD** [comment]
thanks for the comments, i've replaced print() by logging.info() as suggested in a new pull request.
let me know if you have guidance on when to use info() vs error() etc.

**Comment by michaelarnauts** [comment]
You can just add commits on your branch instead of creating a new merge request.

**Pull request closed without merging**